android/app/build.gradle

```diff
@@ -138,7 +138,7 @@ android {
 
 dependencies {
     compile project(':react-native-picker')
-    compile project(':react-native-smart-splashscreen')
+    compile project(':react-native-smart-splash-screen')
```

Comment:
What is this for?

Reply:
Makes errors on android when that is not renamed

Reply:
could you paste the error?

Reply:
Keep git checking out way way back and I still cannot reproduce the error. Well it started back then that there is a react-native-smart-splashscreen and react-native-smart-splash-screen on the build.gradle. Thing is the package name is "react-native-smart-splash-screen". It it is react-native-smart-splashscreen it will throw an error.